test/mocha/cli.js

```diff
@@ -0,0 +1,21 @@
+//var UglifyJS = require('../../');
+var assert = require("assert");
+var exec = require("child_process").exec;
+
+describe("bin/uglifyjs", function() {
+    it("should produce a functional build when using --self", function (done) {
+        exec(__dirname + '/../../bin/uglifyjs --self -cm --wrap WrappedUglifyJS', function (err, stdout) {
```

Comment:
Won't the following work just as well? Might avoid issues with crazy Windows paths.
```
+        exec('../../bin/uglifyjs --self -cm --wrap WrappedUglifyJS', function (err, stdout) {
```